src/tests/integration/test-data/foundry-prove-all

```diff
@@ -71,6 +71,9 @@ BytesTypeTest.testFail_bytes32(bytes32)
 BytesTypeTest.testFail_bytes4(bytes4)
 ChainIdTest.test_chainid_setup()
 CoinBaseTest.test_coinbase_setup()
+ConstructorTest.test_run_constructor_false()
+ConstructorTest.testFail_run_constructor_false()
+ConstructorTest.run_constructor()
```

Comment:
Would `run_constructor()` work and have the same output when initialization code is not executed?